main-actions/src/main/scala/sbt/Sync.scala

```diff
@@ -26,42 +26,44 @@ import sjsonnew.{ Builder, JsonFormat, Unbuilder, deserializationError }
  * It is safe to use for its intended purpose: copying resources to a class output directory.
  */
 object Sync {
-  def apply(store: CacheStore, inStyle: FileInfo.Style = FileInfo.lastModified, outStyle: FileInfo.Style = FileInfo.exists): Traversable[(File, File)] => Relation[File, File] =
-    mappings =>
-      {
-        val relation = Relation.empty ++ mappings
-        noDuplicateTargets(relation)
-        val currentInfo = relation._1s.map(s => (s, inStyle(s))).toMap
-
-        val (previousRelation, previousInfo) = readInfo(store)(inStyle.format)
-        val removeTargets = previousRelation._2s -- relation._2s
-
-        def outofdate(source: File, target: File): Boolean =
-          !previousRelation.contains(source, target) ||
-            (previousInfo get source) != (currentInfo get source) ||
-            !target.exists ||
-            target.isDirectory != source.isDirectory
-
-        val updates = relation filter outofdate
-
-        val (cleanDirs, cleanFiles) = (updates._2s ++ removeTargets).partition(_.isDirectory)
-
-        IO.delete(cleanFiles)
-        IO.deleteIfEmpty(cleanDirs)
-        updates.all.foreach((copy _).tupled)
-
-        writeInfo(store, relation, currentInfo)(inStyle.format)
-        relation
-      }
+  def apply(store: CacheStore,
+            inStyle: FileInfo.Style = FileInfo.lastModified,
+            outStyle: FileInfo.Style = FileInfo.exists)
+    : Traversable[(File, File)] => Relation[File, File] =
+    mappings => {
+      val relation = Relation.empty ++ mappings
+      noDuplicateTargets(relation)
+      val currentInfo = relation._1s.map(s => (s, inStyle(s))).toMap
+
+      val (previousRelation, previousInfo) = readInfo(store)(inStyle.format)
+      val removeTargets = previousRelation._2s -- relation._2s
+
+      def outofdate(source: File, target: File): Boolean =
+        !previousRelation.contains(source, target) ||
+          (previousInfo get source) != (currentInfo get source) ||
+          !target.exists ||
+          target.isDirectory != source.isDirectory
+
+      val updates = relation filter outofdate
+
+      val (cleanDirs, cleanFiles) = (updates._2s ++ removeTargets).partition(_.isDirectory)
+
+      IO.delete(cleanFiles)
+      IO.deleteIfEmpty(cleanDirs)
+      updates.all.foreach((copy _).tupled)
+
+      writeInfo(store, relation, currentInfo)(inStyle.format)
+      relation
+    }
 
   def copy(source: File, target: File): Unit =
     if (source.isFile)
       IO.copyFile(source, target, true)
     else if (!target.exists) // we don't want to update the last modified time of an existing directory
-    {
-      IO.createDirectory(target)
-      IO.copyLastModified(source, target)
-    }
+      {
```

Comment:
This is also weird, you want `else if (...) {` on a single line, not indenting the block as if it was a normal expression.